### PR TITLE
Remove queryCatalog and getFolderContents skin scripts

### PR DESCRIPTION
**Pull request by jensens**
- part of https://github.com/plone/Products.CMFPlone/issues/3515 
- see also https://github.com/plone/Products.CMFPlone/pull/3949#pullrequestreview-2061606187

### CODE REVIEW

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
First merge https://github.com/plone/plone.app.content/pull/281
then this one.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
check fields in https://github.com/plone/Products.CMFPlone/issues/1801 when finished

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/browser/syndication/adapters.py [hunk 1]
```diff
@@ -154,6 +154,7 @@ def language(self):
 
 class CollectionFeed(FolderFeed):
     def _brains(self):
+        catalog = getToolByName(self.context, "portal_catalog")
```
Comment (yurj) [inline Products/CMFPlone/browser/syndication/adapters.py:157]: here catalog is not used.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**